.github/workflows/release.yaml

```diff
@@ -31,7 +33,7 @@ env:
   # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
   USE_GIT_LFS: false
 
-{% raw -%}
+{%- raw -%}
```

Comment:
Curious what this `raw` block does?

Reply:
It means that the variables that come later are not `liquid` variables, and thus should not be expanded by `cargo generate`

Reply:
The raw block just means: "Don't modify what's in this block". If I don't include it, then nothing will work as the syntax for actions conflicts with the templating syntax.

Reply:
Got it, thanks.